microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -10,13 +10,17 @@ ms.date: 10/24/2022
 ---
 # Windowed vs. visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+There are three options for hosting the Microsoft Edge WebView2 control in your app:
+* Windowed Hosting
+* Window to Visual Hosting
+* Visual Hosting
 
-If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article.  If you want to provide a more custom user experience (UX), and want to use visual hosting, read this article.
+If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article. If you want to provide a more custom user experience (UX) and want to use visual hosting, or if you are using windowed mode in specific scenarios and are experiencing persistent issues with DPI and scaling, read this article.
```

Comment:
the term "windowed mode" is used here w/o introduction, and is different than the intro'd term "Windowed hosting".

context includes: i'm checking consistent cap'zn of the 3 terms

Reply:
result:
```
or if you are using windowed mode (Windowed hosting) in specific scenarios

A combination of windowed mode (Windowed hosting) and visual mode (Visual hosting).
```